Update Hosting documentation (#8510)
Scille/parsec-cloud: docs/hosting/introduction.rst

.. _doc_hosting_intro:

> Hosting Introduction
> ====================

Reviewer: ```suggestion
Introduction
============
```

> This section is dedicated to the hosting of Parsec. It will guide you through the deployment, maintenance and administration of the Parsec server and its components.

Reviewer: I'll keep it short here
```suggestion
This guide describes Parsec server deployment, maintenance and administration.
```
Or maybe a slightly more "friendly":
```suggestion
This guide will help you to deploy, maintain and manage your own Parsec server.
```
